frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareFleetMaintained/FleetMaintainedAppDetailsPage/FleetMaintainedAppDetailsPage.tsx

```diff
@@ -196,7 +197,20 @@ const FleetMaintainedAppDetailsPage = ({
     } catch (error) {
       // quick exit if there was an error adding the software. Skip the policy
       // creation.
-      renderFlash("error", getErrorMessage(error));
+
+      const ae = (typeof error === "object" ? error : {}) as AxiosResponse;
+      if (
+        ae.status === 408 ||
+        ae.status === 500 ||
```

Comment:
Seems like we shouldn't have 500s in this bucket, as there are other reasons for 500s to happen beyond timeouts that should be dumped into the generic "couldn't add" bucket. 408 and JSON decoder error make sense though.